packages/hawtio/src/ui/page/HawtioHeader.tsx

```diff
@@ -104,9 +105,27 @@ const HawtioHeaderToolbar: React.FunctionComponent = () => {
     userItems.pop()
   }
 
+  /*
+   * Determine which plugin is currently displaying
+   * based on the path of the current location
+   */
+  const pluginFromLocation = (): Plugin | null => {
+    const path = location.pathname
+    if (!path) return null
```

Comment:
when can it be `!path`?  since `pathname` is of type `string`, this check seems useless.